Matlab_Scripts/5_Depth_Colourless/Study/VR_P3_Depth_Colourless_ERP.m

```diff
@@ -0,0 +1,237 @@
+%% Plot ERPs of your epochs
```

Comment:
Ok so there are a lot of different .m files, and as a naive viewer of the code I don't know which order to run them in to do what.

You can create a master.m file that runs each of these codes with a comment about what they do, 

```
%This script plots the ERPs 
VR_P3_Depth_Colourless
%This script plots the topographies
...
```